tcms/kiwi_auth/forms.py

```diff
@@ -56,6 +57,7 @@ def save(self, commit=True):
     def set_activation_key(self):
         return UserActivationKey.set_random_key_for_user(user=self.instance)
 
+    @override(settings.LANGUAGE_CODE)
     def send_confirm_mail(self, request, activation_key):
```

Comment:
this already calls `mailto()` which overrides the language. Why do you need to override it again ?

Reply:
I use `override` here because subject is translated in this function before sending to `mailto()`